python/res/enkf/model_config.py

```diff
@@ -16,17 +16,35 @@
 from cwrap import BaseCClass
 
 from ecl.summary import EclSum
-
+from ecl.util.util import StringList
 from res import ResPrototype
-from res.job_queue import ForwardModel
+from res.job_queue import ForwardModel, ExtJob, ExtJoblist
 from res.sched import HistorySourceEnum
 from res.util import PathFormat
+from res.enkf import ConfigKeys
+from res.enkf.util import TimeMap
+
+import os
 
 
 class ModelConfig(BaseCClass):
     TYPE_NAME = "model_config"
 
     _alloc                       = ResPrototype("void*  model_config_alloc( config_content, char*, ext_joblist, int, ecl_sum)", bind=False)
+    _alloc_full = ResPrototype("void*  model_config_alloc_full( int, \
```

Comment:
Yet another interesting white space indentation, but this one there might be hope for...